app/models/article.rb

```diff
@@ -1,3 +1,4 @@
 class Article < ApplicationRecord
   validates :title, :body, :author, presence: true
+  belongs_to :user
```

Comment:
Shouln't it belong to a research group? I could be wrong here though

Reply:
This needs to be long to a user with the role of research group.

Reply:
Research group is just a role of the user, so the Article belongs to the user